all.yml

```diff
@@ -5,7 +5,7 @@
   environment: "{{ proxy_env }}"
   tasks:
     - import_role:
-        name: confluent.variables
+        name: variables
 
     - name: Create Certificate Authority and Copy to Ansible Host
```

Comment:
This doesn't need to be tackled here, but it'd be beneficial for the collections cause just to move the certificate_authority tasks file into the ssl role, and call it with a import_role - tasks_from because tasks files are not going to work in collections

Reply:
True, but we are probably not able to pull in playbooks from the collection before ansible-core 2.11 is out. So I suggest doing that as a separate task on the main branch. Could that go into 6.1.x?

Reply:
That doesnt have to hold this up, but it will later when we come against the playbook issue. lets leave it, thats not the goal of this PR